src/migration/telemetry-module-user-configuration-migration.js

```diff
@@ -0,0 +1,46 @@
+import appRootPath from 'app-root-path';
+import path from 'path';
+import BaseMigration from './base-migration.js';
+
+class TelemetryModuleUserConfigurationMigration extends BaseMigration {
+    async executeMigration() {
+        const configurationFolderPath = path.join(appRootPath.path, '..');
+        const configurationFilePath = path.join(
+            configurationFolderPath,
+            this.config.configFilename,
+        );
+
+        const userConfiguration = await this.fileService.readFile(configurationFilePath, true);
+
+        let newTelemetryConfig;
+
+        if ('telemetry' in userConfiguration) {
+            const oldConfigTelemetry = userConfiguration.telemetry;
+            newTelemetryConfig = {
+                enabled: oldConfigTelemetry.enabled,
+                implementation: {
+                    'ot-telemetry': {
+                        enabled: oldConfigTelemetry.enabled,
+                        package: './telemetry/implementation/ot-telemetry.js',
+                        config: {
+                            enabled: oldConfigTelemetry.enabled,
```

Comment:
We don't need enabled flag inside implementation config, we have global flag enabled